blazor/dropdown-list/value-binding.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title: Value Binding in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Value Binding in Syncfusion Blazor DropDown List component and more.

```

Comment:
Description: Checkout and learn here all about Value Binding in Syncfusion Blazor DropDown List component and more.